src/SWP/Bundle/CoreBundle/Resources/config/serializer/Model.Revision.yml

```diff
@@ -0,0 +1,2 @@
+SWP\Bundle\CoreBundle\Model\Revision:
+    exclusion_policy: All
```

Comment:
new line is missing and you should not expose all fields by default, expose manually needed fields